src/routes/example_route.ts

```diff
@@ -0,0 +1,35 @@
+import pool from '../dbconfig/dbconfig';
+import Route from '../types/route';
+
+
+
+const exampleRoute:Route = ['/','GET',  (req:Request,res:any)=>{
+    //it should be wrapped in a try catch;
+    try {
+        const sql = "SELECT * FROM test";
```

Comment:
also we should avoid using direct queries, as I recall we are using ```prisma``` as an ORM

Reply:
agreed. I am working on getting prisma to work with the docker container. (I have been running into issues with the client generation - still looking into it) - this was just a working test: it will be updated once prisma is up and running :)